### PR TITLE
mopidy: update to 1.0.0

### PR DESCRIPTION
**Pull request by allan**

### CODE REVIEW

File: srcpkgs/mopidy/template [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'mopidy'
 pkgname=mopidy
-version=0.19.5
+version=1.0.0
 revision=1
 build_style=python-module
```
Comment (asergi) [inline srcpkgs/mopidy/template]: Revision must be set to 1.

File: srcpkgs/mopidy/template [hunk 2]
```diff
@@ -8,8 +8,7 @@ maintainer="allan <mail@may.mooo.com>"
 license="Apache-2.0"
 homepage="http://www.mopidy.com"
 distfiles="${PYPI_SITE}/M/Mopidy/Mopidy-${version}.tar.gz"
-checksum=940604eadd6435ebf7a85bbc0f15ea57a7fdf5e7e276da17732955ab10050627
-
+checksum=509ec28f176ceea5dc213c16069763ba5496744bf96e117c292fee38f5007fee
 noarch=yes
 python_versions=2.7
```
Comment (asergi) [inline srcpkgs/mopidy/template]: By default, python_versions is set to 2.7, so you can remove it.

Comment (asergi) [inline srcpkgs/mopidy/template]: You can remove python-devel from both makedepends and hostmakedepends.
Also, python-pykka from hostmakedepends, too.

File: srcpkgs/mopidy/template (hunk 1 above)
Comment (dominikh) [inline srcpkgs/mopidy/template]: Revision *is* set to 1?

**Pull request closed without merging**